src/target/drivers/display/spi/128x64x1_common.h

```diff
@@ -79,41 +57,27 @@ void LCD_Contrast(unsigned contrast)
 {
     //int data = 0x20 + contrast * 0xC / 10;
     LCD_Cmd(0x81);
-    int c = LCD_CONTRAST_FUNC(contrast);
-    LCD_Cmd(c);
+    if (HAS_OLED_DISPLAY) {
+        contrast = _oled_contrast_func(contrast);
```

Comment:
why not hide the contract difference in the display driver and expose single function to upper layer?

Reply:
I'm not following you, that is exactly what this is.  there s a single function LCD_Contrast() which needs to decide whether to pick the lcd or oled path

Reply:
somehow I believe I see this _lcd_contrast_func earlier in the code change when I was reviewing. you are absolute right.